nvm.sh

```diff
@@ -859,7 +859,6 @@ nvm_get_arch() {
   case "$NVM_UNAME" in
     *x86_64*) NVM_ARCH=x64 ;;
     *i*86*) NVM_ARCH=x86 ;;
-    *armv6l*) NVM_ARCH=arm-pi ;;
```

Comment:
This will then conflict with `node.js` - see http://nodejs.org/dist/v0.10.0/ for example, which uses only "arm-pi" and not "armv61" or "armv71".

Any solution to this for `io.js` must continue to work on `node.js`.